.circleci/config.yml

```diff
@@ -276,22 +400,100 @@ jobs:
             curl --location --request POST $tsystems_upload_url \
             --header "Authorization: Bearer $tsystems_upload_bearer" \
             --form "file=@${fileName}" \
+
+  instrumentation_tests_device:
+    macos:
+      xcode: "12.3.0"
+    resource_class: large
+    steps:
+      - checkout
+      - setup-android-macos
+      - run-gradle-cmd:
+          desc: Build Apks for instrumentation test
+          cmd: >
+            :Corona-Warn-App:assembleDeviceDebug
+            :Corona-Warn-App:assembleDeviceDebugAndroidTest
+      - run-emulator-for-api:
+          apilevel: 29
+      - run-gradle-cmd:
+          desc: Run instrumentation tests on device
+          cmd: >
+            :Corona-Warn-App:connectedDeviceDebugAndroidTest
+            --info --continue
+            -Pandroid.testInstrumentationRunnerArguments.notAnnotation=testhelpers.Screenshot
+            -Pandroid.testInstrumentationRunnerArguments.clearPackageData=true
+      #      - kill-all-emulators
+      #      - run-emulator-api23
+      #      - run-gradle-cmd:
+      #          desc: Run instrumentation test command
+      #          cmd: |
+      #            :Corona-Warn-App:connectedDeviceForTestersDebugAndroidTest \
+      #            --info \
+      #            --continue \
+      #            -Pandroid.testInstrumentationRunnerArguments.notAnnotation=testhelpers.Screenshot \
+      #            -Pandroid.testInstrumentationRunnerArguments.clearPackageData=true
```

Comment:
Please remove it if not needed anymore?

Reply:
I want to later add API23 tests, but you are right, we can re-add it later.